site/src/templates/exercise.jsx

```diff
@@ -70,7 +78,15 @@ export const Head = ({ pageContext }) => {
 
 export const exerciseQuery = graphql`
   query ($id: String!, $category: String!) {
+    allDirectory(filter: { relativeDirectory: { eq: "exercises" } }) {
+      nodes {
+        base
+      }
```

Comment:
Get all base directories within `/exercises` to determine the available languages.